fdbclient/rapidjson/document.h

```diff
@@ -2041,7 +2041,7 @@ class GenericValue {
 #elif RAPIDJSON_64BIT
 		char payload[sizeof(SizeType) * 2 + sizeof(void*) + 6]; // 6 padding bytes
 #else
-		    char payload[sizeof(SizeType) * 2 + sizeof(void*) + 2]; // 2 padding bytes
+		  char payload[sizeof(SizeType) * 2 + sizeof(void*) + 2]; // 2 padding bytes
```

Comment:
```suggestion
		char payload[sizeof(SizeType) * 2 + sizeof(void*) + 2]; // 2 padding bytes
```

Reply:
I dont have an issue with this change, (i think it looks better) but it breaks the format check, I am reverting it.

Reply:
That's weird, the introduction of two spaces here shouldn't be what our spec would want. We can merge it and then maybe I'll try playing around with it some.

Reply:
Looks like it's getting confused by the preprocessor macros and applying the `ConstructorInitializerIndentWidth` setting here. Not sure if there's much we can do about that.